webui/README.md

```diff
@@ -53,12 +53,23 @@ The content are described in the template file.
 You will need to change the defaults if you would like to enable authentication or are running the eidos on
 a port other than 8080.
 
-### Running the Server
+### Running the Server for Development
+
+To build for dev mode, run:
+
+```bash
+turbo run build
+```
+
+Or run a file watcher so that you do not need to rebuild when updating components.
+```bash
+turbo run watch
```

Comment:
watch is not in the package.json for webui--should the dev be running this from eidolon-ui-2?

Reply:
I added this command to turbo.json. I believe it picks up the dependencies transitively since it was working ootb since it was updating packages for me, but I definitely not sure if this is the right way to do it.